org.opentosca.bus.management.api.soaphttp/src/org/opentosca/bus/management/api/soaphttp/processor/RequestProcessor.java

```diff
@@ -83,6 +83,9 @@ public void process(final Exchange exchange) throws Exception {
             }
         }
 
+        // add header with public IP of engine-ia for the raspbian pull IA
+        exchange.getIn().setHeader("OPENTOSCA_PUBLIC_IP", Settings.OPENTOSCA_PUBLIC_IP);

```

Comment:
Follow-up from above; AFAIK this is the public hostname of the IA engine, right? So, we also should call the constant accordingly.